schema.sql

```diff
@@ -0,0 +1,68 @@
+
+-- Table for movies
+CREATE TABLE movies (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  title VARCHAR(255),
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label VARCHAR(255),
+  publish_date DATE,
+  archived BOOLEAN
+);
+
+-- Table for sources
+CREATE TABLE sources (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(255) 
+)
```

Comment:
As requested by the previous reviewer, kindly remove the remove `movies` and `sources` tables, they are required (only if the group has 4 members). 🤓💪

Since you are 3 in your group, all tasks should be based on the cards from your [Kanban board](https://github.com/MuhammadUsmanAslam/catalog_of_my_things/projects/1).